### PR TITLE
Capstone 3

### PR DESCRIPTION
**Pull request by JuanLPalacios**
Implementation of a mobile web application to check a list of narrativa API COVID-19 metrics.
[here is a video](https://www.loom.com/share/303de3707e7d47c8908e0e24bdffa56e) with my opinions on it.

### CODE REVIEW

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @JuanLPalacios, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
  - [x] As the project requires, Kindly recorded a video presenting the features of the project.
      - The maximum length of the video is 5 minutes.
      - The video contains a demo of the project, highlighting all the project features.
      - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
  - [x] The project requires implementing unit tests for several components of the app and some pure functions in Redux code.
  - [x] The project requires to create some sort of a filter on home page. If it is not possible to implement category filter depending on  topic of your project, you can implement search instead. Kindly do that.
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by JuanLPalacios** [comment]
the link to the video is on the PR description

File: src/views/Countries.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    [date],
+  );
+  const prevYear = new Date(new Date().setFullYear(date.getFullYear() - 1));
+  const nextYear = new Date(new Date().setFullYear(date.getFullYear() + 1));
+  const prevMonth = new Date(new Date().setMonth(date.getMonth() - 1));
+  const nextMonth = new Date(new Date().setMonth(date.getMonth() + 1));
+  const prevDay = new Date(new Date().setDate(date.getDate() - 1));
+  const nextDay = new Date(new Date().setDate(date.getDate() + 1));
+  const prevValue = VALUE_LIST[
```
Comment (julie-ify) [inline src/views/Countries.js:26]: - [ ] Nice filter implementation 👍

**Review by julie-ify — APPROVED** [review]
Hi @JuanLPalacios, 

## Highlights!
- Tests passed ✔️ 
- Good video presentation ✔️ 
- Filter implementation works ✔️
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## APPROVED ✔️ ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**